File: SiteVisitsNetCore2020Web/ClientApp/src/app/models/SearchEngine.ts
export class SearchEngine {
  //id: number;
  id: string;
  name: string;
  host: string;

  constructor(options: {
    //id: number,
    id: string,
    name: string,
    host: string
  }) {
    if (options) {
      this.id = options.id;
      this.name = options.name;
      this.host = options.host;
    }
  }
    
}

File: SiteVisitsNetCore2020Web/Models/SearchEngine.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace SiteVisitsNetCore2020Web.Models
{
    public class SearchEngine
    {
        //public int Id { get; set; }
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Host { get; set; }
        public DateTime FirstOccurrence { get; set; }
        public DateTime DateCreated { get; set; }

    }
}
